include/voltorb_flip/voltorb_flip_data.h

```diff
@@ -25,8 +25,8 @@ typedef struct BgTemplates {
     BgTemplate unk0[6];
 } BgTemplates;
 
-const u8 sMainMenuMsgNos[];
-const u8 sGameInfoMsgNos[];
-const u8 ov122_021E9270[];
+extern const u8 sMainMenuMsgNos[];
```

Comment:
do these need to be defined in the header or can we move them to the c file?

Reply:
Splitting voltorb_flip.c and merging them with the respective voltorb_flip_data*.c is out-of-scope for this PR.